src/core/Akka.Cluster/Member.cs

```diff
@@ -192,10 +196,13 @@ internal class AddressComparer : IComparer<Address>
             /// <inheritdoc cref="IComparer{Address}.Compare"/>
             public int Compare(Address x, Address y)
             {
-                if (x.Equals(y)) return 0;
-                if (!x.Host.Equals(y.Host)) return String.Compare(x.Host.GetOrElse(""), y.Host.GetOrElse(""), StringComparison.Ordinal);
-                if (!x.Port.Equals(y.Port)) return Nullable.Compare(x.Port.GetOrElse(0), (y.Port.GetOrElse(0)));
```

Comment:
There's no sense is all those Equals comparisons, they are redundant in context of comparison (especially `x.Equals(y)` which basically is comparison made twice here).